2-run-kobiton-test.md

```diff
@@ -0,0 +1,86 @@
+# Run Kobiton Automation Test on Travis CI
+This guild will teach you how to run your Kobiton test on Travis and how to secure your secret enviroment variables.
+
+## 2.1 Prepare your Kobiton username and API key for Automation Testing
+### 2.1.1 Go to Kobiton Portal and click on your Account Name on the upper right corner
+  - You can see your username on the dialog below.
+![](assets/2_kobiton_username.jpg)
+### 2.1.2 Click on your Account Name and select "Settings"
+  - Copy your API key from the textbox.
+![](assets/2_kobiton_apikey.jpg)
+
+## 2.2 Configure enviroment variables in your .travis.yml file
+### 2.2.1 There are 3 methods to attach an enviroment variable to travis.
+  - If it is public and is the same on every branches, add it to your .travis.yml
+  - If it contains sensitive information (i.e API Key) but is the same on every branches, add it to Travis CI Repository Settings
+  - If it contains sensitive information (i.e API Key) and might be different for different branches, encrypt it first and add to your .travis.yml
+
+In this guide, we will add our Username and API key to our .travis.yml file. The API Key will be encrypted for security purpose.
```

Comment:
purposes
".travis.yml" -> just "travis.yml"
I don't know if you need that dot at the beginning but it looks kinda weird like that